pages/secure/sec_graylog_beats.rst

```diff
@@ -0,0 +1,102 @@
+*******************************
+Secured Graylog and Beats input
+*******************************
+
+The goal of this guide is to have a secured Graylog interface, API and secure communication for Beats that are authenticated by certificate. This way only trusted sources are able to deliver messages into Graylog. 
+
+This is a structured document that contains only information already given at various location in this documentation. It should give the missing connection between the different parts of the documentation.
+
+
+SSL/TLS prework
+===============
+
+Create a CA with our `shadowCA <https://github.com/graylog-labs/shadowCA>`__ or use your already given CA. That is needed to create all certificates. The examples will take the given names from our shadowCA and reference to that only, please adjust this to your local needs. If in doubt check the shadowCA scripts what kind of certificate is created and used.
+
+The CA certificate needs to be imported on all machines that are part of the setup using the `documented steps <https://github.com/graylog-labs/shadowCA/blob/master/docs/add_ca_to_truststore.md>`__. Depending on your Browser you might need to add the ``.der`` to your Browser to trust the CA. In addition the CA ``.der`` file is added to a JVM Keystore that is used by Graylog.
```

Comment:
Use `imported` instead of `add`. I was confused and thought about the file extension.

Reply:
nice finding - with your words in mind that is really more helpful!